model/quic-bbr.cc

```diff
@@ -549,16 +551,17 @@ QuicBbr::WhichState (BbrMode_t mode) const
 {
   switch (mode)
     {
-    case 0:
-      return "BBR_STARTUP";
-    case 1:
-      return "BBR_DRAIN";
-    case 2:
-      return "BBR_PROBE_BW";
-    case 3:
-      return "BBR_PROBE_RTT";
+      case 0:
+        return "BBR_STARTUP";
+      case 1:
+        return "BBR_DRAIN";
+      case 2:
+        return "BBR_PROBE_BW";
+      case 3:
+        return "BBR_PROBE_RTT";
```

Comment:
The correct way to handle this is to add a default case to the switch, which triggers the error